api/pom.xml

```diff
@@ -3,7 +3,7 @@
 	<parent>
 		<groupId>org.openmrs.module</groupId>
 		<artifactId>patientflags</artifactId>
-		<version>1.3.5-SNAPSHOT</version>
+		<version>1.4.5-SNAPSHOT</version>

```

Comment:
Don't worry about changing the version... you only increment the version number after doing an official maven release.